test/tests/datepicker-dialog.js

```diff
@@ -282,41 +287,372 @@ ariaTest('', exampleFile, 'dialog-shift-tab', async (t) => {
   }
 });
 
-// ariaTest('', exampleFile, 'month-year-button-space-return', async (t) => {
-// });
+ariaTest('ENTER to buttons change calendar and date in focus', exampleFile, 'month-year-button-space-return', async (t) => {
+  t.plan(4);
 
-// ariaTest('', exampleFile, 'grid-space-return', async (t) => {
-// });
+  await t.context.session.findElement(By.css(ex.buttonSelector)).click();
+  // By default, focus will be on todays date.
+  let day = new Date();
+
+  // send enter to next month button
+
+  await t.context.session.findElement(By.css(ex.nextMonthButton)).sendKeys(Key.ENTER);
+  day.setMonth(day.getMonth() + 1);
+  let dayInFocus = await t.context.session
+    .findElement(By.css(ex.currentlyFocusedButton))
+    .getAttribute('data-date');
+
+  t.is(
+    dayInFocus,
+    day.toISOString().split('T')[0],
+    'After selected next month button, date should be ' + day.toISOString().split('T')[0] + ' but found: ' + dayInFocus
+  );
+
+  // send enter to next year button
+  await t.context.session.findElement(By.css(ex.nextYearButton)).sendKeys(Key.ENTER);
+  day.setFullYear(day.getFullYear() + 1);
+  dayInFocus = await t.context.session
+    .findElement(By.css(ex.currentlyFocusedButton))
+    .getAttribute('data-date');
+
+  t.is(
+    dayInFocus,
+    day.toISOString().split('T')[0],
+    'After selected next month button, then next year button date should be ' + day.toISOString().split('T')[0] + ' but found: ' + dayInFocus
+  );
+
+  // Send enter to previous month button
+  await t.context.session.findElement(By.css(ex.prevMonthButton)).sendKeys(Key.ENTER);
+  day.setMonth(day.getMonth() - 1);
+  dayInFocus = await t.context.session
+    .findElement(By.css(ex.currentlyFocusedButton))
+    .getAttribute('data-date');
+
+  t.is(
+    dayInFocus,
+    day.toISOString().split('T')[0],
+    'After selected next month button, then next year button date, then previous month button, date should be ' + day.toISOString().split('T')[0] + ' but found: ' + dayInFocus
+  );
+
+  // Send enter to previous year button
+
+  await t.context.session.findElement(By.css(ex.prevYearButton)).sendKeys(Key.ENTER);
+  day.setFullYear(day.getFullYear() - 1);
+  dayInFocus = await t.context.session
+    .findElement(By.css(ex.currentlyFocusedButton))
+    .getAttribute('data-date');
+
+  t.is(
+    dayInFocus,
+    day.toISOString().split('T')[0],
+    'After selected next month button, then next year button date, then previous month button, then previous year button, date should be ' + day.toISOString().split('T')[0] + ' but found: ' + dayInFocus
+  );
+
+  // TODO: Test SPACE
```

Comment:
Is it better to do `t.skip` instead of a comment? Or does that cause weirdness with the coverage scripts

Reply:
Oh, I see there actually is a todo https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#test-placeholders-todo

Reply:
Hmm I dug into it more yesterday thinking it might have been a webdriver bug (because I couldn't replicate it) -- but then I realized it's a firefox only bug. I added a commit with the failing tests and made a issue: https://github.com/w3c/aria-practices/issues/1098